branch_submit.go

```diff
@@ -21,7 +21,7 @@ import (
 // submitOptions defines options that are common to all submit commands.
 type submitOptions struct {
 	DryRun bool `short:"n" help:"Don't actually submit the stack"`
-	Fill   bool `help:"Fill in the change title and body from the commit messages"`
+	Fill   bool `short:"f" help:"Fill in the change title and body from the commit messages"`
```

Comment:
`f` has risk of being confused for `--force`, which currently doesn't have a short flag, but could conceivably get one.

Instead of `f` as short for fill, how does `F` or `C` (as in "use my commits") sound?

Reply:
I'm fine with `-F`. That said, `--force` seems like the more dangerous of the options, so should that be the one with the capital letter? (i.e. we wanna have a speed bump with `--force`, so we'd want it's short option to be the one that takes a little longer to type).

Reply:
I agree—`--force` should have a speed bump. But:

- Git uses `-f`/`--force`. Matching Git isn't a hard requirement, but it's a matter of convention and expectations.
- `-f` as short for `--force` is reasonably common in other commonly used commands, including `rm`.
- There's already `-f`/`--force` on `branch delete`, so short form here will probably want to match.

I think I'm in favor of dropping the short form `-f` from `branch delete` because a) the operation is dangerous, so the speedbump, and b) that command will prompt you for a force-like behavior anyway, if needed. (I'll do that separately from this PR.)

But I'm still having trouble using `-f` for something not `--force` given how prevalent that short form is for that flag. Looking at `git commit` for inspiration:

- `-C <commit>` takes the new commit message from the given commit
- `-c <commit>` takes the new commit message from the given commit and opens an editor
- `-F <file>` takes the commit message from a file

While we don't take an argument with --fill, any of those three seem much more digestible here.